process/process_areas/quality_management/guidance/quality_work_product_review_guideline.rst

```diff
@@ -0,0 +1,45 @@
+..
+   # *******************************************************************************
+   # Copyright (c) 2025 Contributors to the Eclipse Foundation
+   #
+   # See the NOTICE file(s) distributed with this work for additional
+   # information regarding copyright ownership.
+   #
+   # This program and the accompanying materials are made available under the
+   # terms of the Apache License Version 2.0 which is available at
+   # https://www.apache.org/licenses/LICENSE-2.0
+   #
+   # SPDX-License-Identifier: Apache-2.0
+   # *******************************************************************************
+
+
+Quality Work Product Review Guideline
+=====================================
+
+.. gd_guidl:: Quality work product review
+   :id: gd_guidl__wp_review
+   :status: valid
+
+Purpose
+-------
+| Definition of a generic workflow to ensure the quality of work products.
+
+Objectives
+----------
+| Check and assure the quality of the project/platform work products.
+
+Process
+-------
+
+| Step  1: Create issue with label ``q_wp_review`` in status OPEN
+| Step  2: Identify which work products will be checked during the review and document it in the created issue
+| Step  3: Adjust :need:`gd_chklst__review_checklist` based on specified work products to review, which contains acceptance criteria for all work products
+| Step  4: Create PR, Link the stored checklist in the PR to the issue
```

Comment:
Here I have a disconnect as the steps require different roles. Who is doing this, Isn't the work product already created with another ticket? Shouldn't it be more like that the stored checklist is linked to the PR where the work product is handled, which you require to decide which work product you are taking?